src/python/pants/jvm/goals/coursier_integration_test.py

```diff
@@ -75,9 +88,9 @@ def test_creates_missing_lockfile(rule_runner: RuleRunner) -> None:
     result = rule_runner.run_goal_rule(CoursierResolve, args=[], env_inherit={"PATH"})
     assert result.exit_code == 0
     assert result.stderr == "Updated lockfile at: 3rdparty/jvm/default.lock\n"
-    assert (
-        Path(rule_runner.build_root, "3rdparty/jvm/default.lock").read_bytes()
-        == HAMCREST_EXPECTED_LOCKFILE.to_serialized()
+    compare_lockfiles(
```

Comment:
This changes the test -- we're now deserializing the file on disk and comparing to the structure we defined in the test, rather than checking that the structure is byte-for-byte correct.

Reply:
Much better :)